### DISCUSSION TITLE
Marigold depth - high-quality diffusion-based monocular depth estimation (drop-in alternative)

### DISCUSSION THREAD

**Discussion by toshas** [Mikubill/sd-webui-controlnet#2318] [Ideas]
We have just released a novel relative depth estimation method, which is derived from stable diffusion. It delivers unprecedented crispness and works well in the wild. It should be fairly straightforward to integrate it. I would expect better control when conditioning on these depth maps:
https://marigoldmonodepth.github.io/
https://twitter.com/AntonObukhov1/status/1732946419663667464

**Comment by lllyasviel** [comment]
Marigold and patchfusion should be looked at

**Comment by kirby-jack** [comment]
hi there, what controlnet model do you think would pair well with the Marigold??
